pages.es/common/amass-viz.md

```diff
@@ -0,0 +1,24 @@
+# amass viz
+
+> Visualize gathered information in a network graph.
+> Más información: <https://github.com/OWASP/Amass/blob/master/doc/user_guide.md#the-viz-subcommand>.
+
+- Genera una visualización D3.js basada en los datos de la base de datos:
+
+`amass viz -d3 -dir {{ruta/a/directorio_de_base_de_datos}}`
```

Comment:
```suggestion
- Genere una visualización D3.js basada en datos específicos de la base de datos::

`amass viz -d3 -dir {{ruta/a/directorio_de_base_de_datos}}`
```

Optional fix (error source: original page): unexpected `database` term, `specific` expected.
Note: `Generate a D3.js visualization based on specific database data:` can be used as a translation source.

Reply:
The current translation with `Genera` is ok.
the DB is specified, no the data. `basada en datos de una específica base de datos` is the correct semantic. but leave as-is.